ansible_builder/main.py

```diff
@@ -22,9 +32,21 @@ def __init__(self, filename='execution-environment.yml', base_image=default_base
     def version(self):
         return self.definition.version
 
-    def process(self):
+    def create(self):
         return self.containerfile.write()
 
+    def build(self):
+        self.create()
+        command = [self.container_runtime, "build"]
+        arguments = ["-f", self.containerfile.path,
+                     "-t", self.tag,
+                     self.build_context]
+        for arg in arguments:
+            command.append(arg)
+        result = subprocess.run(command, capture_output=True)
```

Comment:
I would love to have a test that asserts that `command` contains certain elements, whether or not we override things like the build context path or the tag. We shouldn't actually be invoking `docker build` in our unit tests, so we'll want to move the command construction into a separate method that can be tested in isolation.